syntax = "proto3";

package bakdata.trade.v1;

message Trade {
  string id = 1;
  string personid = 2;
  string issuerid = 3;
  string companyid = 4;
  string position = 5;
  string asset_type = 6;
  string trade_type = 7;
  double avg_price = 8;
  string avg_price_currency = 9;
  double aggregate_volume = 10;
  string aggregate_volume_currency = 11;
  string date_of_notification = 12;
  string date_of_trade = 13;
  string place_of_trade = 14;
  string date_of_activation = 15;
  string issuer_name = 16;
}